tools/packaging/nfpm.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+if [[ -z "${NFPM_SIGNING_KEY_FILE}" ]]; then
+  echo "NFPM_SIGNING_KEY_FILE is not set"
+  exit 1
+fi
+if [[ -z "${NFPM_PASSPHRASE}" ]]; then
+  echo "NFPM_PASSPHRASE is not set"
+  exit 1
+fi
+
+rm -rf dist/tmp && mkdir -p dist/tmp/packages
+
+for name in pyroscope profilecli; do
+  for arch in amd64 arm64; do
```

Comment:
Those arch should be enough for the users running on baremetal. This is similar to mimir.